ui/src/app/shared/header/header.component.ts

```diff
@@ -198,6 +198,19 @@ export class HeaderComponent implements OnInit, AfterViewInit, AfterViewChecked,
     this.search();
   }
 
+  hasChipsToClear(): boolean {
+    return this.getChipKeys().filter(key => key != 'projectId').length > 0;
```

Comment:
Just a curious question, why exclude `projectId ` from the query keys? Is that an invisible but fixed key for the query builder?

Reply:
It was part of the request (see #388); I'm assuming it's because if it's being used (possibly only for dsub?) it's not likely to change while a user is looking at jobs. @kbergin ?

Reply:
I do believe the reasoning for this was related to dsub. For dsub when you log in as a certain project it would put a project id chip in the query builder, but if you close out that chip I think it logs you out of the job manager. I'm not sure if that still holds true, my information is probably outdated :)